apiary.apib

```diff
@@ -954,11 +954,570 @@ Outputs the list of peers that were removed.
 
 # Group config
 
-## edit
+## config [POST /config{?arg1,arg2}{&bool,json}]
```

Comment:
Should be POST, however, both work because the http-api in go-ipfs doesn't care about the HTTP method.